fdbserver/DataDistribution.actor.cpp

```diff
@@ -2926,27 +2918,34 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 		    .detail("DesiredTeamsPerServer", SERVER_KNOBS->DESIRED_TEAMS_PER_SERVER);
 	}
 
-	// Adds storage servers held on process of which the Process Id is “pid” into excludeServers which prevent
+	// Adds storage servers held on process of which the Process Id is “id” into excludeServers which prevent
 	// recruiting the wiggling storage servers and let teamTracker start to move data off the affected teams;
 	// Return a vector of futures wait for all data is moved to other teams.
-	std::vector<Future<Void>> excludeStorageServersForWiggle(const Value& pid) {
-		std::vector<Future<Void>> moveFutures;
-		if (this->pid2server_info.count(pid) != 0) {
-			for (auto& info : this->pid2server_info[pid]) {
-				AddressExclusion addr(info->lastKnownInterface.address().ip, info->lastKnownInterface.address().port);
-				if (this->excludedServers.count(addr) &&
-				    this->excludedServers.get(addr) != DDTeamCollection::Status::NONE) {
-					continue; // don't overwrite the value set by actor trackExcludedServer
-				}
+	Future<Void> excludeStorageServersForWiggle(const UID& id) {
+		Future<Void> moveFuture = Void();
+		if (this->server_info.count(id) != 0) {
+			auto& info = server_info.at(id);
+			AddressExclusion addr(info->lastKnownInterface.address().ip, info->lastKnownInterface.address().port);
+
+			// don't overwrite the value set by actor trackExcludedServer
```

Comment:
misplaced comment?

Reply:
This comment explains the following code: prevent the `WIGGLING` state from overriding other abnormal states (like FAILED, EXCLUDE).